### DISCUSSION TITLE
Only WebApps?

### DISCUSSION THREAD

**Discussion by Symbolics** [rabbibotton/clog#81] [General]
I can see that clog is very well suited to building web apps, and kudos to the author.

How well does this fit the model of a development environment? Say if someone wanted to reproduce something like the LispWorks GUI? For example a package browser, debugger, etc? Or launch a window with some content from a REPL (ideally an emacs slime REPL). As I look at the examples, it seems like this might be a square peg in a round hole.

**Comment by rabbibotton** [comment]
Take a look at demo 3 and you already have a great head start :) I think that it is an excellent fit for it.

**Comment by aykaramba** [comment]
I have been trying to figure out how to load up LEM with CLOG.  In the desktop example, there is a text editor component that I have been playing around with to run *.lisp files which is really cool.  I can imagine re-purposing that component to handle keyboard input + rendering of the ncurses interface directly in a web page.  If I can learn enough, I'd love to create a CLOG front end for LEM.

There is so much that is possible when the lines between local and remote are blurred.
